entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityQueryServiceImpl.java

```diff
@@ -230,12 +230,21 @@ Row convertToEntityQueryResult(
                           .setValueType(ValueType.STRING)
                           .setString(entity.getEntityName())
                           .build());
+                } else if (edsSubDocPath.equals(EntityServiceConstants.ENTITY_CREATED_TIME)) {
```

Comment:
so if edsSubDocPath is equal to this, that means we put in an explicit ES mapping between some attribute ID and this path?

Reply:
discussed offline, some mapping is required so reasonable to use the subdocpath map which is used for v1 entity id and names. V2 treats these as attributes as they're determined by the writer, but should already support using the mapping for this case.

An ideal but more difficult solution in terms of compatibility would be to store the creation timestamp attribute id on the entity types and then only write to there, but out of scope for now.